tests/integration/tier0/destructive/conversion-method/test_custom_repos.py

```diff
@@ -37,9 +37,9 @@ def test_run_conversion_using_custom_repos(shell, convert2rhel):
     with open("/etc/system-release", "r") as file:
         system_release = file.read()
         system_version = get_system_version(system_release_content=system_release)
-        if system_version.major == 7:
-            enable_repo_opt = "--enablerepo rhel-7-server-rpms --enablerepo rhel-7-server-optional-rpms --enablerepo rhel-7-server-extras-rpms"
-        elif system_version.major == 8:
+        enable_repo_opt = "--enablerepo rhel-7-server-rpms --enablerepo rhel-7-server-optional-rpms --enablerepo rhel-7-server-extras-rpms"
```

Comment:
This one is not critical and can be merged as is, but style-wise it concerns me.

I'll show you on another example.

Let's assume we want to set variable `manager` to `dnf` on CentOS and to `apt` on Ubuntu

Ex. 1
```
if system == "centos":
  # on centos
  manager = "dnf"
else:
  # assuming ubuntu
  manager = "apt"
```

This works within the current assumptions that we only have centos and ubuntu. It will fail if some day, 3 years from now, we run on SUSE

So the more future-proof variant would be:
Ex. 2
```
if system == "centos":
  # on centos
  manager = "dnf"
elif system == "ubuntu":
  # on ubuntu
  manager = "apt"
else:
  logger.critical("Unsupported system", system)
```

But there is more. The Pythonic way of  doing the same would be:
Ex. 3
```
MANAGERS = {
    "centos": "dnf",
    "ubuntu": "apt",
} 

manager = MANAGERS[system]
```

In that Ex. 3 we clearly state that there is no real branching in the program logic. We simply define the facts based on the current system.

This 3 approach is not always easy to implement so depending on the situation I use Ex 2 or Ex 3. But I would avoid Ex 1 because it leads to errors appearing somewhere down the line, and they are not easy to track.